codegen-sbt/src/main/scala/caliban/codegen/CalibanSourceGenerator.scala

```diff
@@ -11,6 +11,27 @@ object CalibanSourceGenerator {
   import zio._
   import zio.console._
 
+  import sjsonnew.{ :*:, LList, LNil }
```

Comment:
Is this a dependency of sbt?

Reply:
Yeah, built in already. There is one downside about this approach, where if sbt breaks bincompat, it means a new release will need to be cut (or newer versions of the plugin will not be compatible with some older versions of sbt).

I haven't found a good solution for this, and this seems to be the only way to implement this functionality, so I'm at a loss other than to just raise awareness about it.